Build script to export all notebooks

artste opened the pull request:
Simple build script to export all the notebooks in a row and update *.py files in "exp" folder.

jph00 commented:
To make this cross-platform @artste , how about instead adding a "--all" flag in the python script itself?

artste commented:
Great! I'll do it.

artste commented:
TL;DR

Export in literal order all notebooks whose file name is "under" string '06'.

```shell
fastai_docs/dev_course/dl2$ ./notebook2script.py --upTo=06
Begin...
Converted 00_exports.ipynb to exp/nb_00.py
Converted 01_matmul.ipynb to exp/nb_01.py
Converted 02_fully_connected.ipynb to exp/nb_02.py
Converted 02a_why_sqrt5.ipynb to exp/nb_02a.py
Converted 02b_initializing.ipynb to exp/nb_02b.py
Converted 03_minibatch_training.ipynb to exp/nb_03.py
Converted 04_callbacks.ipynb to exp/nb_04.py
Converted 05_anneal.ipynb to exp/nb_05.py
Converted 05a_foundations.ipynb to exp/nb_05a.py
Converted 05b_early_stopping.ipynb to exp/nb_05b.py
...End
```

Export in literal order all notebooks.
```shell
fastai_docs/dev_course/dl2$ ./notebook2script.py --upTo=06
```
